src/components/dropDown/DropDown.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react';
+import { ScrollView, StyleSheet, Text, View, ViewStyle } from 'react-native';
+import { List } from 'react-native-paper';
+import { useTheme } from '../../styles/themes';
+import { Icon } from '../icons/Icon';
+
+export type DropDownOption = {
+    name: string;
+    value: any;
+};
+
+interface Props {
+    disabled?: boolean;
+    error?: boolean;
+    placeholder: string;
+    label?: string;
```

Comment:
I'm not sure either what this label correspond to. Maybe we should clarify that with the PrDs

Reply:
okay, i will ask them

Reply:
I checked with Hélène, we don't need the label option here : all we display is part of the options list

Reply:
thanks!